docs/guide/site_factory.md

```diff
@@ -286,6 +286,24 @@
 
     User Group skeletons are retained after deleting the site.
     
+    ### Define Roles to be automatically updated when a site is created  
+    
+    Role definitions can contain user/login policies with limitations that limit user access to certain sites. 
+    To avoid the need to revisit all Roles and add the newly created SiteAccess to a list of limitations, you can decide that the Roles you select are automatically updated when the site is created.
```

Comment:
Or even better:
```suggestion
    To avoid the need to add the new SiteAccess to Limitations for all Roles,
    you can decide that the Roles you select are automatically updated when the site is created.
```